airflow/www/views.py

```diff
@@ -3667,6 +3667,19 @@ class AirflowModelView(ModelView):
 
     CustomSQLAInterface = wwwutils.CustomSQLAInterface
 
+    def __getattribute__(self, attr):
+        attribute = object.__getattribute__(self, attr)
+        if callable(attribute) and attr in [
+            "add",
+            "edit",
+            "delete",
+            "download",
+            "action",
+            "action_post",
+        ]:
+            return action_logging(event=f"{self.route_base.strip('/')}.{attr}")(attribute)
+        return attribute
```

Comment:
If the name list is definite, can we simply override those methods, instead of doing this magic function?

Reply:
The problem with overriding is how to generate the event name at the decorator level. `{self.route_base.strip('/')}.{attr}` we won't have the `self` to recognize if the action is for variable or connection

Reply:
Let’s add a docstring to explain this as well. We should also add something to the _class’s docstring_ since readers might not be able to pick up `__getattribute__`.